python/py-fypp/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-fypp
+version             3.1
+revision            0
+categories-append   devel
+license             BSD
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Python-powered Fortran preprocessor
+long_description    Fypp is a Python-powered preprocessor. It can be used for any programming languages \
+                    but its primary aim is to offer a Fortran preprocessor, which helps to extend Fortran \
+                    with condititional compiling and template metaprogramming capabilities. \
+                    Instead of introducing its own expression syntax, it uses Python expressions \
+                    in its preprocessor directives, offering the consistency and versatility of Python \
+                    when formulating metaprogramming tasks. It puts strong emphasis on robustness \
+                    and on neat integration into developing toolchains.
+homepage            https://github.com/aradi/fypp
+checksums           rmd160  ede50d1312a8669d5adb7a3735d37e8c1971e3f8 \
+                    sha256  b777ae70081c7c82edcf52eefc01a3e8a566b4e82ccd33ca3b2f06faf1d07d36 \
+                    size    68237
+
+python.versions     36 37 38 39 310 311
```

Comment:
please no old Python versions unless needed by other packages; I suggest "310 311" only

Reply:
I would retain 39 too, as for the other port, for compatibility. Are we sure python310 works on every old MacOS? I am not. But the bigger issue is that there are packages without 310+, which potentially creates a problem.

Reply:
310 works everywhere, and 311 too (after Evan’s PR gets committed for G3 PPC support)

Reply:
@kencu @reneeotten Moved to 311 for `dbcsr`, dropped old pythons for `py-fypp`.